### PR TITLE
Update Funnel UI to MUI (material design css framework)

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: funnel/templates/session_form.html.jinja2 [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+{% block content %}
+	<form id="{{ formid }}" method="POST" accept-charset="UTF-8" class="mui-form">
+	  {{ renderform_inner(form, formid or None) }}
+	</form>
+{% endblock %}
+
+{% block footerscripts %}
+	<script>
+	  SHOW_SAVE_BUTTON = true;
+	</script>
```
Comment (jace) [inline funnel/templates/session_form.html.jinja2]: If you have a script tag without content type like this, how does it work?

File: funnel/templates/venues.html.jinja2 [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
-        {{ venue.description }}
+        <div class="mui--text-body2">{{ venue.description }}</div>
         {%- if venue.rooms %}
           <ol>
             {%- for room in venue.rooms %}
               <li>
                 <p>
-                  <a href="{{ room.url_for('edit') }}">{{ room.title }}</a>
-                  <a href="{{ room.url_for('delete') }}"><i class="fa fa-trash"></i></a>
+                  <span class="mui--text-subhead"><strong>{{ room.title }}</strong></spam>&nbsp;
```
Comment (jace) [inline funnel/templates/venues.html.jinja2]: Typo?

File: funnel/views/space.py [hunk 3]
```diff
@@ -194,10 +194,10 @@ def rsvp_list(profile, space):
     ((ProposalSpace, ProposalSpaceRedirect), {'name': 'space', 'profile': 'profile'}, 'space'),
     permission='edit-space')
 def space_transition(profile, space):
-    transitionform = ProposalSpaceTransitionForm(obj=space)
-    if transitionform.validate_on_submit():  # check if the provided transition is valid
+    transition_form = ProposalSpaceTransitionForm(obj=space)
+    if transition_form.validate_on_submit():  # check if the provided transition is valid
         transition = getattr(space.current_access(),
-                             transitionform.transition.data)
+                             transition_form.transition.data)
```
Comment (jace) [inline funnel/views/space.py]: Limit indentation to four spaces.

File: funnel/templates/delete.html.jinja2 [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
-  <input class="btn btn-danger" type="submit" name="delete" value="{% trans %}Delete{% endtrans %}"/>
-  <input class="btn btn-default" type="submit" name="cancel" value="{% trans %}Cancel{% endtrans %}"/>
-</form>
-
+	<p class="mui--text-title">{{ message }}</p>
+	<form id="delete" method="POST" class="mui-form">
+	  <input type="hidden" name="form.id" value="delete"/>
+	  {{ form.hidden_tag() }}
+	  {% if form.csrf_token.errors %}
+	    {% for error in form.csrf_token.errors %}<div class="error">{{ error }}</div>{% endfor %}
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/templates/delete.html.jinja2]: Fixed

File: funnel/templates/session_form.html.jinja2 (hunk 1 above)
Comment (vidya-ram) [reply funnel/templates/session_form.html.jinja2]: from html5, missing `type` attribute indicates the script is JavaScript
But I will add the `type` attribute where ever it is missing.

File: funnel/templates/index.html.jinja2 [hunk 5]
```diff
@@ -7,45 +7,75 @@
 
 {% block description %}{% if g.profile %}{{ g.profile.description|striptags }}{% else %}{{ config['WELCOME_MESSAGE'] }}{% endif %}{% endblock %}
 
-{% block baseheadline -%}
-  {% block headline -%}{%- endblock %}
-{%- endblock %}
+{% block banner %}
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/templates/index.html.jinja2]: Will fix it as part of PR(https://github.com/hasgeek/funnel/pull/254)

**Pull request merged**